modules/apps/portal-security/portal-security-content-security-policy/src/main/java/com/liferay/portal/security/content/security/policy/internal/constants/ContentSecurityPolicyConstants.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.security.content.security.policy.internal.constants;
+
+/**
+ * @author Olivér Kecskeméty
+ */
+public class ContentSecurityPolicyConstants {
+
+	public static final String CONTENT_SECURITY_POLICY_HEADER =
+		"Content-Security-Policy";
+
+	public static final String CONTENT_SECURITY_POLICY_NONCE_HEADER_PREFIX =
+		"nonce-";
+
+	public static final String CONTENT_SECURITY_POLICY_NONCE_TOKEN =
+		"[nonceToken]";
+
+}
```

Comment:
I am not sure about the correct name for this class. Did you find a pattern for introduction of new HTTP headers?

I've found `com.liferay.portal.kernel.servlet.HttpHeaders` and `com.liferay.portal.kernel.security.auth.http.HttpAuthorizationHeader` in `portal-kernel`, but no good examples in modules.

Did you find anything else?

Reply:
Sadly not. I decided to use this because `*Constants.java` is an existing pattern used in many different modules and as it was not required I didn't wanted to change anything in `portal-kernel` and keep everything CSP related under this module.

Reply:
That's fine then. I think we can try with the current name and see what BChan thinks 👍